shards/shards_test.go

```diff
@@ -447,10 +469,10 @@ func TestShardedSearcher_List(t *testing.T) {
 			ignored := []cmp.Option{
 				cmpopts.EquateEmpty(),
 				cmpopts.IgnoreFields(zoekt.RepoListEntry{}, "IndexMetadata"),
-				cmpopts.IgnoreFields(zoekt.RepoStats{}, "IndexBytes"),
```

Comment:
out of interest, why do we not need to ignore this anymore? I think the reason we ignored it before is because it is so sensitive to changes we do in other parts that it is frustrating to have to always update.

Reply:
Alright, I thought we were just not interested in stats before. I'll add back this ignore field.